pages/release-notes.mdx

```diff
@@ -65,6 +65,28 @@ Memgraph (v2.11.1 and lower).
 
 </Callout>
 
+## Lab v2.11.0 - Jan 3, 2023
+
+{<h3> New features and improvements </h3>}
+
+- You can import CSV files with the CSV import configuration
+  tool without writing a single Cypher query.
+- The query plan chart is now available in the summary
+  tab for each query run.
+- You can profile a query plan to enrich the query plan
+  chart with the absolute and relative execution time
+  spent on each query plan step.
+- You can download generated query plan in JSON format.
+- LLM GraphChat now works even when connected to Neo4j.
+- Query data view will be shown by default if there are
+  non-graph data in the query results.
+
+{<h3> Bug fixes </h3>}
+
+- Hiding nodes on graph view now work as expected.
+- Lists in the notifications are now shown with correct
+  indentation.
```

Comment:
```suggestion
- Import CSV files using the CSV import configuration
  tool instead of writing Cypher queries.
- The query plan chart is now available in the *Summary*
  tab for each query run.
- You can profile a query plan to enrich the query plan
  chart with the absolute and relative execution time
  spent on each query plan step.
- The generated query plan can be downloaded in the JSON format.
- LLM GraphChat now works even when connected to Neo4j.
- Query data view will be shown by default if there are
  non-graph data in the query results.

{<h3> Bug fixes </h3>}

- Hiding nodes in the *Graph results* view now works as expected.
- Lists in the notifications are now shown with correct
  indentation.
```